WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -248,19 +248,43 @@ class PostCoordinator: NSObject {
 
 extension PostCoordinator: Uploader {
     func resume() {
-        let service = PostService(managedObjectContext: mainContext)
-
-        service.getFailedPosts { [weak self] posts in
+        let fetcher = FailedPostsFetcher(mainContext)
+        fetcher.getPostsToRetry { [weak self] posts in
             guard let self = self else {
                 return
             }
 
-            posts.forEach() { post in
-                let shouldRetry = post.status == .draft && !post.hasRemote()
+            posts.forEach({ self.retrySave(of: $0) })
```

Comment:
style nitpick, could be just `posts.forEach { self.retrySave(of: $0) }`

Reply:
Thanks for this. Done in e18bb15645d45205b9da4eed99baa44ed08c1dc5.